src/main/java/minicraft/core/io/Localization.java

```diff
@@ -48,7 +48,7 @@ public static String getLocalized(String key) {
 		if (key.matches("^[ ]*$")) return key; // Blank, or just whitespace
 		
 		try {
-			double num = Double.parseDouble(key);
+			Double.parseDouble(key);
```

Comment:
Is this necessary?

Reply:
Technically not _necessary_, but removing unnecessary variable declarations/assignments is probably a good idea.